package/MDAnalysis/core/topology.py

```
@@ -518,12 +518,16 @@ def del_TopologyAttr(self, topologyattr):
     @property
     def guessed_attributes(self):
         """A list of the guessed attributes in this topology"""
-        return filter(lambda x: x.is_guessed, self.attrs)
+        return filter(lambda x: x.is_guessed
+                      if(not isinstance(x.is_guessed, list))
```

Relying on the specific `list` type is a bit fragile. If we ever change `is_guessed` to return a tuple or a numpy array, then this test will fail silently.

A less fragile approach would be to test that `x.is_guessed` is an instance of `typing.Container`. `Container` is an "abstract base class" (or ABC); every object that implements the `__contains__` method (so all the objects with which you can use `in`) will appear as an instance of `Container`.